### PR TITLE
Influxctl 2.0

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md [hunk 1]
```diff
@@ -32,24 +32,213 @@ influxctl [flags] [command]
 
 | Command                                                                 | Description                            |
 | :---------------------------------------------------------------------- | :------------------------------------- |
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:34]: @sanderson @powersj Looking at `cli.go`, `influxctl login` no longer exists? Just curious - looks like we never documented it to begin with.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md [hunk 2]
```diff
@@ -61,26 +61,38 @@ influxctl database create [--retention-period 0s] <DATABASE_NAME>
...
+| :--- | :------------------- | :----------------------------------------------------------- |
+|      | `--retention-period` | Database retention period (default is 0s or infinite)        |
+|      | `--max-tables`       | Maximum tables per database (default is 500, 0 uses default) |
+|      | `--max-columns`      | Maximum columns per table (default is 250, 0 uses default)   |
+| `-h` | `--help`             | Output command help                                          |
 
 ## Examples
 
 - [Create a database with an infinite retention period](#create-a-database-with-an-infinite-retention-period)
 - [Create a database with a 30 day retention period](#create-a-database-with-a-30-day-retention-period)
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md:74]:
```suggestion
- [Create a database with a 30-day retention period](#create-a-database-with-a-30-day-retention-period)
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md [hunk 3]
```diff
@@ -61,26 +61,38 @@ influxctl database create [--retention-period 0s] <DATABASE_NAME>
...
 
-##### Create a database with an infinite retention period
+### Create a database with an infinite retention period
 
 ```sh
 influxctl database create mydb
 ```
 
-##### Create a database with a 30 day retention period
+### Create a database with a 30 day retention period
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md:83]:
```suggestion
### Create a database with a 30-day retention period
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/token/get.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl token
+weight: 301
+---
+
+The `influxctl token get` command returns information about a database token
+in an InfluxDB Cloud Dedicated cluster.
+
+The `--format` option lets you print result in other formats.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/token/get.md:15]:
```suggestion
The `--format` option lets you print the output in other formats.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/token/list.md [hunk 5]
```diff
@@ -12,6 +12,10 @@ weight: 301
 The `influxctl token list` command lists all database tokens in an InfluxDB Cloud
 Dedicated cluster.
 
+The `--format` option lets you print result in other formats.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/token/list.md:15]:
```suggestion
The `--format` option lets you print the output in other formats.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/token/create.md [hunk 6]
```diff
@@ -37,11 +37,11 @@ influxctl token create \
 
 ## Flags
 
-| Flag |                    | Description                           |
-| :--- | :----------------- | :------------------------------------ |
-| `-h` | `--help`           | Output command help                   |
+| Flag |                    | Description                                        |
+| :--- | :----------------- | :------------------------------------------------- |
 |      | `--read-database`  | Grant read permissions to a database. Repeatable.  |
 |      | `--write-database` | Grant write permissions to a database. Repeatable. |
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/token/create.md:43]:
```suggestion
|      | `--read-database`  | Grant read permissions to a database _(Repeatable)_ |
|      | `--write-database` | Grant write permissions to a database _(Repeatable)_ |
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/user/list.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl user
+weight: 301
+---
+
+The `influxctl user list` command lists all users associated with your InfluxDB
+Cloud Dedicated account ID.
+
+The `--format` option lets you print result in other formats.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/user/list.md:14]:
```suggestion
The `--format` option lets you print the output in other formats.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/get.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl cluster
+weight: 301
+---
+
+The `influxctl cluster get` command returns information about an InfluxDB
+Cloud Dedicated cluster.
+
+The `--format` option lets you print result in other formats.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/get.md:15]:
```suggestion
The `--format` option lets you print the output in other formats.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/list.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl cluster
+weight: 301
+---
+
+The `influxctl cluster list` command returns information about all InfluxDB
+Cloud Dedicated clusters associated with your account ID.
+
+The `--format` option lets you print result in other formats.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/list.md:15]:
```suggestion
The `--format` option lets you print the output in other formats.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/list.md [hunk 10]
```diff
@@ -12,6 +12,10 @@ weight: 301
 The `influxctl database list` command lists all databases in an InfluxDB Cloud
 Dedicated cluster.
 
+The `--format` option lets you print result in other formats.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/list.md:15]:
```suggestion
The `--format` option lets you print the output in other formats.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Looks good, minor fixes.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:34]: Yeah, I think it only existed in one version and it never did get added.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/get.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+| :--- | :--------- | :-------------------------------------------- |
+|      | `--format` | Output format (`table` _(default)_ or `json`) |
+| `-h` | `--help`   | Output command help                           |
+
+## Examples
+
+##### Return information about a cluster
+
+```sh
+influxctl cluster get 000xX0Xx00xX
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/get.md:43]: The cluster IDs are UUIDs so maybe use that as an example here? or say `<cluster id>` or similar?

**Review by powersj — COMMENTED** [review]
Thank you for doing this, couple questions/comments

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md [hunk 12]
```diff
@@ -32,24 +32,213 @@ influxctl [flags] [command]
...
+
+1.  **Download the `influxctl` CLI package** appropriate for your CPU type.
+    Download the package from your browser or command line.
+
+    ##### Browser {#macos-browser-download}
+
+    <a class="btn download" href="https://dl.influxdata.com/influxctl/releases/influxctl-v{{< latest-influxctl >}}-darwin-x86_64.zip" download>influxctl CLI v{{< latest-influxctl >}} (x86_64)</a>
+    <a class="btn download" href="https://dl.influxdata.com/influxctl/releases/influxctl-v{{< latest-influxctl >}}-darwin-arm64.zip" download>influxctl CLI v{{< latest-influxctl >}} (arm64)</a>
+
+    ##### Command line {#linux-command-line-download}
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:75]:
```suggestion
    ##### Command line {#macos-command-line-download}
```

[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:75]: Yes it should. Nice catch.

**Review by powersj — APPROVED** [review]
Thank you very much!

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md [hunk 13]
```diff
@@ -32,24 +32,214 @@ influxctl [flags] [command]
...
+1.  **Download the `influxctl` CLI package**.
+
+    <a class="btn download" href="https://dl.influxdata.com/influxctl/releases/influxctl-v{{< latest-influxctl >}}-windows-x86_64.zip" download>influxctl CLI v{{< latest-influxctl >}} (x86_64)</a>
+
+2.  **Expand the downloaded archive**.
+
+    Expand the downloaded archive into C:\Program Files\InfluxData\ and rename it if desired.
+
+    ```powershell
+    > Expand-Archive .\influxctl-v{{< latest-influxctl >}}-windows-x86_64.zip -DestinationPath 'C:\Program Files\InfluxData\'
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:219]:
```suggestion
    Expand-Archive .\influxctl-v{{< latest-influxctl >}}-windows-x86_64.zip `
    -DestinationPath 'C:\Program Files\InfluxData\'
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md [hunk 14]
```diff
@@ -32,24 +32,214 @@ influxctl [flags] [command]
...
+    > mv 'C:\Program Files\InfluxData\influxctl-v{{< latest-influxctl >}}-windows-x86_64' 'C:\Program Files\InfluxData\influxctl'
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:220]:
```suggestion
    mv 'C:\Program Files\InfluxData\influxctl-v{{< latest-influxctl >}}-windows-x86_64' `
    'C:\Program Files\InfluxData\influxctl'
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md (hunk 13 above)
Comment (jstirnaman) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:219]: Use backtick for line continuation in PowerShell.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md [hunk 15]
```diff
@@ -59,22 +249,84 @@ following credentials:
 - InfluxDB Cloud Dedicated account ID
 - InfluxDB Cloud Dedicated cluster ID
 
-Use the [`influxctl init` command](/influxdb/cloud-dedicated/reference/cli/influxctl/init)
-to start an interactive prompt that creates and stores the required credentials
-as a **connection profile**.
+### Create a configuration file
+
+Create a `config.toml` that includes the necessary credentials and store in
+on your machine.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:255]:
```suggestion
Create a `config.toml` that includes the necessary credentials.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
+---
+title: influxctl database update
+description: >
+  The `influxctl database update` command updates databases' retention period,
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md:4]:
```suggestion
  The `influxctl database update` command updates a database's retention period,
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+  tables, or columns.
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl database
+weight: 301
+---
+
+The `influxctl database update` command updates databases' retention period,
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md:12]:
```suggestion
The `influxctl database update` command updates a database's retention period,
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+| :--- | :------------------- | :----------------------------------------------------------- |
+|      | `--retention-period` | Database retention period (default is 0s or infinite)        |
+|      | `--max-tables`       | Maximum tables per database (default is 500, 0 uses default) |
+|      | `--max-columns`      | Maximum columns per table (default is 250, 0 uses default)   |
+| `-h` | `--help`             | Output command help                                          |
+
+## Examples
+
+- [Update a database's retention period](#update-a-databases-retention-period)
+- [Update a databases's table limit](#update-a-databasess-table-limit)
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md:39]:
```suggestion
- [Update a database's table limit](#update-a-databases-table-limit)
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md [hunk 19]
```diff
@@ -0,0 +1,86 @@
...
+- [Update a databases's column limit](#update-a-databasess-column-limit)
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md:40]:
```suggestion
- [Update a database's column limit](#update-a-databases-column-limit)
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md [hunk 20]
```diff
@@ -0,0 +1,86 @@
...
+- `6w`: 6 weeks
+- `1mo`: 1 month (30 days)
+- `1y`: 1 year
+- `30d30d`: 60 days
+- `2.5d`: 60 hours
+
+{{% /flex-content %}}
+{{< /flex >}}
+
+### Update a databases's table limit
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md:76]:
```suggestion
### Update a database's table limit
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md [hunk 21]
```diff
@@ -0,0 +1,86 @@
...
+
+```sh
+influxctl database update mydb --max-tables 300
+```
+
+### Update a databases's column limit
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/update.md:82]:
```suggestion
### Update a database's column limit
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/user/delete.md [hunk 22]
```diff
@@ -0,0 +1,49 @@
+---
+title: influxctl user delete
+description: >
+  The `influxctl user delete` command ...
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/user/delete.md:4]:
```suggestion
  The `influxctl user delete` command deletes a user from your InfluxDB Cloud Dedicated account.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/user/list.md [hunk 23]
```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl user list
+description: >
+  The `influxctl user list` command ...
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/user/list.md:4]:
```suggestion
  The `influxctl user list` command lists all users associated with your InfluxDB
Cloud Dedicated account ID.
```

**Review by jstirnaman — APPROVED** [review]
Minor fixes and PowerShell formatting.

**Pull request merged**